include/tables/scene_table.h

```diff
@@ -110,7 +114,12 @@
 /* 0x62 */ DEFINE_SCENE(spot18_scene, g_pn_41, SCENE_GORON_CITY, SDC_GORON_CITY, 0, 0)
 /* 0x63 */ DEFINE_SCENE(spot20_scene, g_pn_42, SCENE_LON_LON_RANCH, SDC_LON_LON_RANCH, 0, 0)
 /* 0x64 */ DEFINE_SCENE(ganon_tou_scene, g_pn_43, SCENE_OUTSIDE_GANONS_CASTLE, SDC_OUTSIDE_GANONS_CASTLE, 0, 0)
-#if DEBUG_ASSETS
+
+#if !DEBUG_ASSETS && defined(DEFINE_SCENE_ENUM)
+           DEFINE_SCENE_TABLE_END()
+#endif
+
+#if DEBUG_ASSETS || defined(DEFINE_SCENE_ENUM)
```

Comment:
This `#if` logic makes the table unnecessarily hard to read, I'd prefer if we just had `DEFINE_SCENE_DEBUG` and `DEFINE_SCENE_DELETED` and defined them at all include sites to keep the table itself easier to read